src/routes/users/fetchAllUsers.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from "fastify";
+import { User } from "../../types/User";
+import UserModel from "../../models/user";
+
+type FetchAllUsers = {
+  Reply: User[] | { error: { code: string; message: string } };
+};
+
+const url = "/user";
+
+export const handler: RouteHandler<FetchAllUsers> = async (req, reply) => {
+  try {
+    const users = await UserModel.query().select("firstName", "lastName","areaId");
+    if (users.length > 0) {
+      reply.status(200).send(users);
+    } else {
+      reply.status(404).send({
+        error: {
+          code: "unknown",
+          message: "No users found.",
+        },
+      });
+    }
+  } catch (error) {
+    return reply.status(500).send({
+        error: {
+          code: "unknown",
+          message: `An unknown error occurred when trying to fetch users. Error: ${error}`,
+        },
+      });
+  }
+};
+
+export const schema = {
+  operationId: "fetchAllUsers",
+  tags: ["User"],
+  summary: "Fetch All Users",
+  response: {
+    200: {
```

Comment:
if this returns an array of users, we can use the existing User interface on `/types` 😸